src/braket/jobs/hybrid_job.py

```diff
@@ -274,7 +277,7 @@ def __enter__(self):
         for module in self._modules:
             cloudpickle.register_pickle_by_value(module)
 
-    def __exit__(self, exc_type, exc_val, exc_tb):
+    def __exit__(self, exc_type, exc_val, exc_tb):  # noqa: ANN001
```

Comment:
Should be adding types here?

Reply:
This is added as a baseline - these types will be documented later.